felles/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>no.nav.sbl.dialogarena</groupId>
+        <artifactId>modiabrukerdialog</artifactId>
+        <version>dev</version>
+    </parent>
+
+    <artifactId>felles</artifactId>
```

Comment:
Kan vi finne på et bedre navn her? :sweat_smile:
Dette er vel kun ting som er felles for proxy og "core" appen? Som igjen kun er Arena + infotrygd. Kan vi kalle det `arena-infotrygd-models` eller noe lignende som er litt mer beskrivende (og constraining)?